LICENSE

```diff
@@ -2,7 +2,7 @@ UNICODE LICENSE V3
 
 COPYRIGHT AND PERMISSION NOTICE
 
-Copyright © 2016-2024 Unicode, Inc.
+Copyright © 1991-2025 Unicode, Inc.
```

Comment:
@yumaoka : It sounds like, regardless of the confusion about the source of truth, this should be the date range in the ICU copy. Please apply and move on.
```suggestion
Copyright © 2016-2025 Unicode, Inc.
```